railties/test/application/configuration_test.rb

```diff
@@ -283,6 +283,29 @@ def change
       assert_instance_of Pathname, Rails.root
     end
 
+    test "application url is set from config file, available as Rails.application.url and sets default_url_options" do
```

Comment:
Let's make this 2 separate test scenarios, long test descriptions mean having to pass all of that to `-n` when trying to run a single test

Reply:
I will shorten them. To run the test separately, you should use the line number. 
```
bin/test test/application/configuration_test.rb:286
```